packages/contractkit/src/wrappers/MultiSig.ts

```diff
@@ -33,7 +33,11 @@ export class MultiSigWrapper extends BaseWrapper<MultiSig> {
    * Otherwise, submits the `txObject` to the multisig and add confirmation.
    * @param index The index of the pending withdrawal to withdraw.
    */
-  async submitOrConfirmTransaction(destination: string, txObject: TransactionObject<any>) {
+  async submitOrConfirmTransaction(
+    destination: string,
+    txObject: TransactionObject<any>,
+    value: string = '0'
```

Comment:
Does `TransactionObject` not have a `value` field?

Reply:
it has an `arguments` array of type "any", but no direct `value` field